samples/jwt-server/jwt-server.yaml

```diff
@@ -25,9 +25,106 @@ spec:
   - name: http
     port: 8000
     targetPort: 8000
+  - name: https
+    port: 8443
+    targetPort: 8443
   selector:
     app: jwt-server
 ---
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: jwt-server-cert-key
+data:
+  #command to generate certificate and a key
+  #openssl req \ -x509 \ -nodes \ -newkey rsa:2048 \ -keyout server.key \ -out server.crt \ -days 3650 \
```

Comment:
remove the `\` in the command, you are already putting it in a single line, you don't need the `\` any more.

Reply:
sure !!